CHANGES

```diff
@@ -20,7 +20,7 @@ X.Y (2017-12-XX)
   - Convert C functions from K&R to ANSI C
 
 1.5.1 (2016-01-14)
```

Comment:
Please also add a new `1.5.x` entry with a bullet point summarizing your changes, e.g. `- manual converted to GAPDoc`

Reply:
Done.